doc/source/release-notes/release-notes_index.rst

```diff
@@ -1,23 +1,10 @@
 ============================================
- |Percona XtraDB Cluster| 5.6 Release notes
+ |Percona XtraDB Cluster| 5.7 Release notes
 ============================================
 
 .. toctree::
    :maxdepth: 1
    :glob:
 
-   Percona-XtraDB-Cluster-5.6.29-25.15
-   Percona-XtraDB-Cluster-5.6.28-25.14
-   Percona-XtraDB-Cluster-5.6.27-25.13
-   Percona-XtraDB-Cluster-5.6.26-25.12
-   Percona-XtraDB-Cluster-5.6.25-25.12
-   Percona-XtraDB-Cluster-5.6.24-25.11
-   Percona-XtraDB-Cluster-5.6.22-25.8
-   Percona-XtraDB-Cluster-5.6.21-25.8
-   Percona-XtraDB-Cluster-5.6.20-25.7
-   Percona-XtraDB-Cluster-5.6.19-25.6
-   Percona-XtraDB-Cluster-5.6.15-25.5
-   Percona-XtraDB-Cluster-5.6.15-25.4
-   Percona-XtraDB-Cluster-5.6.15-25.3
-   Percona-XtraDB-Cluster-5.6.15-25.2
-   Percona-XtraDB-Cluster-5.6.14-25.1
+   Percona-XtraDB-Cluster-5.7.11-4beta
```

Comment:
I assume the naming convention here is 5.7.11-4beta-25.14.2 .. please re-confirm with RE team.